src/fragments/lib/graphqlapi/js/mutate-data.mdx

```diff
@@ -62,6 +62,10 @@ const deletedTodo = await API.graphql({ query: mutations.deleteTodo, variables:
 
 Only an `id` is needed.
 
+<Callout>
```

Comment:
We currently plan to add this in the CPK docs, wondering if we should align the phrasing here?

https://github.com/aws-amplify/docs/pull/4452/files#diff-4d957a94c2c999b5fb4f2dcb9748be613e4a2849b5893d54d7489199f4de504fR85-R90

Reply:
Updated the wordings to match CPK docs PR.